go-controller/pkg/cni/cni.go

```diff
@@ -116,6 +116,59 @@ func (pr *PodRequest) cmdDel() ([]byte, error) {
 	return []byte{}, nil
 }
 
+func (pr *PodRequest) cmdCheck(podLister corev1listers.PodLister) ([]byte, error) {
+	namespace := pr.PodNamespace
+	podName := pr.PodName
+	if namespace == "" || podName == "" {
+		return nil, fmt.Errorf("required CNI variable missing")
+	}
+	ifaceID := fmt.Sprintf("%s_%s", namespace, podName)
+
+	// Get the IP address and MAC address of the pod
+	annotations, err := getPodAnnotations(pr.ctx, podLister, pr.PodNamespace, pr.PodName)
```

Comment:
Don't get these from the pod; get them from the PrevResult.

Reply:
Nevermind; this isn't correct.

Reply:
Why is it incorrect, it is the same test that ovn-kubernetes does at the end of the ADD command
https://github.com/ovn-org/ovn-kubernetes/blob/1b79b2477bcbd92d42fb86b3c785b742985471cc/go-controller/pkg/cni/helper_linux.go#L375

Reply:
sorry, I meant to say that I was wrong.